packages/user-group-service/src/groups/resolver.ts

```diff
@@ -25,12 +26,13 @@ export const GroupResolver = {
   Group: {
     members ( root: any, GraphQLArgs: any, ctx: any ) {
       return UserGroupAPIHelper.roverFetch( `/groups/${ root.cn }` )
-            .then( ( res ) => {
-              return res.result?.memberUids.map( ( member: string ) => {
-                return member.substring( 4, member.indexOf( ',ou' ) );
-              } );
-            } )
-            .catch((err) => { throw("There is some problem fetching members of the group ") });
+        .then( ( res ) => {
+          const uids = res.result?.memberUids.map( ( member: string ) => {
+            return member.substring( 4, member.indexOf( ',ou' ) );
```

Comment:
A better way to do this would be to split the string at `,` and use the sub-section that starts with `uid`.
The `memberUid` strings are not restricted to a common format of `uid,ou,...`. It can have some other fields between uid and ou. So just make it robust to avoid any future bugs.

Reply:
Done!